snapshot/snapshot.js

```diff
@@ -62,13 +65,13 @@ const main = async () => {
 
   // Validation
 
-  toSnapshot = dedupe(toSnapshot);
   toSnapshot = validate(toSnapshot);
   toSnapshot = lowercase(toSnapshot);
+  toSnapshot = dedupe(toSnapshot);
```

Comment:
felt more natural to validate => lowercase => dedupe rather than dedupe => validate => lowercase